### ISSUE TITLE
Add comenting system

### ISSUE THREAD

**Issue by parthpower** [superuser-blog/superuser#1]
One option is Disqus but it embeds some ads with it. Maybe we can add Facebook comments but not everyone has Facebook or make/find custom commenting system and host it on heroku or something? or gitter?

**Comment by sanketplus** [comment]
let us first finalize on theme because theme installation is not that simple with jekyll. we can embed comments after, possibly disqus.

**Comment by parthpower** [comment]
I want to stay away from disqus because it injects advertisements. Maybe something like https://github.com/imsun/gitment which basically creates an issue for comments or https://mademistakes.com/articles/jekyll-static-comments/ which creates a pull request for a comment and updates posts on approval.

**Comment by sanketplus** [comment]
can we use staticman2?

**Comment by sanketplus** [comment]
also we need to export comments from wordpress

**Comment by parthpower** [comment]
Yeah, it's simple. I did it for test [here](https://github.com/parthpower/minimal-mistakes/blob/c2530d7062291d38cc186f8de396222badff41da/_config.yml#L29)  and working example https://parthpower.github.io/minimal-mistakes/reconfigure-zynq-pl-go/

**Comment by sanketplus** [comment]
I have uncommented the lines. However we need to configure staticman and I cannot seem to be able to do it. (permission issue)

can you add user as collaborator and also put on reCaptcha? @parthpower

**Comment by parthpower** [comment]
You stickman already has write permission and will add reCaptcha.
